charts/airbyte-webapp/templates/secrets.yaml

```diff
@@ -5,13 +5,10 @@ kind: Secret
 metadata:
   name: webapp-secrets
   labels:
-    app.kubernetes.io/name: {{ include "airbyte.name" . }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    helm.sh/chart: {{ include "airbyte.chart" . }}
-    app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
-    app.airbyte.io/fullname: {{ include "airbyte.fullname" . }}
+    {{- include "airbyte.labels" . | nindent 4 }}
   annotations:
+    helm.sh/hook: pre-install,pre-upgrade
```

Comment:
same question here - why do we need this?

Reply:
Same answer as above, to be sure that we have secrets for the deployment before creating the deployment itself(so it'll find those if specified in values.yaml)